print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
+    this.next = next;
+  }
+
+  getValue() {
+    return this.value;
+  }
+
+  setValue(value) {
+    this.value = value;
+  }
```

Comment:
This is also already set in the constructor.  So not necessary?

Reply:
Thank you for the confirmation, I was starting to doubt myself lol

Reply:
I think you can still have a set method if you only want to change one field, but I also don't think it's recommended either way?